suite-common/graph/src/balanceHistoryUtils.ts

```diff
@@ -230,8 +230,9 @@ export const getAccountHistoryMovementItemETH = ({
         }
 
         tx.tokens.forEach(token => {
-            // skip empty amounts and non-ERC20 tokens
-            if (token.amount === '' || token.standard !== 'ERC20') return;
+            // skip empty amounts and non-ERC20 non-BEP20 tokens
```

Comment:
please add comment that BEP is related to polygon

Reply:
b09a27fe8f